ui/App/components/ModsContent.jsx

```diff
@@ -1,121 +1,572 @@
 import React from 'react';
+import ReactDOMServer from 'react-dom/server';
 import {IndexLink} from 'react-router';
-import ModList from './Mods/ListMods.jsx';
-import InstalledMods from './Mods/InstalledMods.jsx';
-import ModPacks from './Mods/ModPacks.jsx'
+import ModOverview from './Mods/ModOverview.jsx';
+import locks from "locks";
 
 class ModsContent extends React.Component {
     constructor(props) {
         super(props);
+
         this.componentDidMount = this.componentDidMount.bind(this);
-        this.toggleMod = this.toggleMod.bind(this);
-        this.loadInstalledModList = this.loadInstalledModList.bind(this);
-        this.loadModPackList = this.loadModPackList.bind(this);
+        this.loadModList = this.loadModList.bind(this);
+        this.handlerFactorioLogin = this.handlerFactorioLogin.bind(this);
+        this.loadDownloadList = this.loadDownloadList.bind(this);
+        this.loadDownloadListSwalHandler = this.loadDownloadListSwalHandler.bind(this);
+        this.toggleModHandler = this.toggleModHandler.bind(this);
+        this.deleteModHandler = this.deleteModHandler.bind(this);
+        this.updateModHandler = this.updateModHandler.bind(this);
+        this.uploadModSuccessHandler = this.uploadModSuccessHandler.bind(this);
+        this.factorioLogoutHandler = this.factorioLogoutHandler.bind(this);
+        this.deleteAllHandler = this.deleteAllHandler.bind(this);
+        this.updateAllModsHandler = this.updateAllModsHandler.bind(this);
+        this.updatesAvailable = this.updatesAvailable.bind(this);
+        this.updateCountSubtract = this.updateCountSubtract.bind(this);
+        this.updateCountAdd = this.updateCountAdd.bind(this);
+
         this.state = {
-            installedMods: [],
-            listMods: [],
-            modPacks: [],
+            logged_in: false,
```

Comment:
Use camelcase here to keep it consistent.

Reply:
I will change to camelcase everywhere :)

Reply:
Do you want to have all ReactJS code in camelCase?

Reply:
Yeah let's make it camelcase here as well.